src/algokit_utils/accounts/kmd_account_manager.py

```diff
@@ -0,0 +1,192 @@
+        """
+        self._client_manager = client_manager
+        try:
+            self._kmd = client_manager.kmd
+        except ValueError:
+            self._kmd = None
+
+    def kmd(self) -> KMDClient:
+        """Get the KMD client, initializing it if needed.
+
+        Returns:
+            KMDClient: The initialized KMD client
+
+        Raises:
+            Exception: If KMD is not configured
+        """
+        if self._kmd is None:
+            if self._client_manager.is_local_net():
+                kmd_config = ClientManager.get_config_from_environment_or_localnet()
+                self._kmd = ClientManager.get_kmd_client(kmd_config.kmd_config)
+                return self._kmd
+            raise Exception("Attempt to use KMD client with no KMD configured")
+        return self._kmd
+
+    def get_wallet_account(
+        self,
+        wallet_name: str,
+        predicate: Callable[[dict[str, Any]], bool] | None = None,
+        sender: str | None = None,
+    ) -> KmdAccount | None:
+        """Returns an Algorand signing account with private key loaded from the given KMD wallet.
+
+        Args:
+            wallet_name: The name of the wallet to retrieve an account from
+            predicate: Optional filter to use to find the account (otherwise returns a random account from the wallet)
+            sender: Optional sender address to use this signer for (aka a rekeyed account)
+
+        Returns:
+            Optional[KmdAccount]: The signing account or None if no matching wallet or account was found
+
+        Example:
+            ```python
+            # Get default funded account in a LocalNet
+            default_dispenser = kmd_manager.get_wallet_account(
+                "unencrypted-default-wallet",
+                lambda a: a["status"] != "Offline" and a["amount"] > 1_000_000_000
+            )
+            ```
+        """
+        kmd_client = self.kmd()
+        wallets = kmd_client.list_wallets()
+        wallet = next((w for w in wallets if w["name"] == wallet_name), None)
+        if not wallet:
+            return None
+
+        wallet_id = wallet["id"]
+        wallet_handle = kmd_client.init_wallet_handle(wallet_id, "")
+        addresses = kmd_client.list_keys(wallet_handle)
+
+        matched_address = None
+        if predicate:
+            for address in addresses:
+                account_info = self._client_manager.algod.account_info(address)
+                if predicate(cast(dict[str, Any], account_info)):
+                    matched_address = address
+                    break
+        else:
+            matched_address = next(iter(addresses), None)
+
+        if not matched_address:
+            return None
+
+        private_key = kmd_client.export_key(wallet_handle, "", matched_address)
+        return KmdAccount(private_key=private_key, address=sender)
+
+    def get_or_create_wallet_account(self, name: str, fund_with: AlgoAmount | None = None) -> KmdAccount:
```

Comment:
```suggestion
    def get_or_create_wallet_account(self, name: str, fund_with: AlgoAmount | None = AlgoAmount.from_algo(1000)) -> KmdAccount:
```

Corrects the log message and also simplifies the code a little.